python/syntax.lark

```diff
@@ -4,75 +4,93 @@ main_instruction: instruction (";" | "\n")
                 | if (";" | "\n")?
                 | ifelse (";" | "\n")?
                 | for (";" | "\n")?
+                | while (";" | "\n")?
                 | for_legacy (";" | "\n")?
 
 last_instruction: instruction (";" | "\n")?
                 | if (";" | "\n")?
                 | ifelse (";" | "\n")?
                 | for (";" | "\n")?
+                | while (";" | "\n")?
                 | for_legacy (";" | "\n")?
 
 instruction: declare
            | function_callback
            | function_callback_pipe
            | imp
            | return
+           | continue
+           | break
            | vary
            | enum_definition
            | alias_definition
            | class_definition
+           | assert
 
 // functions
-vary: "var " NAME "=" expression
-declare: "let " modifier name_type "=" expression
+vary: "var" NAME "=" expression
+declare: "let" modifier name_type "=" expression
```

Comment:
Was this a bug fix for allowing a multiline comment right after let/var? eg `let/* wow */wow=3`

Reply:
This was a bug fix